### ISSUE TITLE
Three-finger tap interferes with the Zoom Accessibility feature of iPhone/iPad, hence does not work

### ISSUE THREAD

**Issue by nikolay-diyanov** [NativeScript/NativeScript#174]
If you have the Accessibility --> Zoom feature of your iDevice enabled, the three-finger tap gesture will not work to LiveSync the iOS app.

**Comment by nikolay-diyanov** [comment]
Logged in the ios-runtime repo.
